pkg/rootless/specconv/specconv_linux.go

```diff
@@ -13,6 +13,18 @@ import (
 	specs "github.com/opencontainers/runtime-spec/specs-go"
 )
 
+// ToRootfulInRootless is used for "rootful-in-rootless" dind;
+// the daemon is running in UserNS but has no access to RootlessKit API socket, host filesystem, etc.
+//
+// This fuction does:
+// * Fix up OOMScoreAdj
```

Comment:
Is it worth leaving a reference here why we're doing this? I'm sure that information gets easily lost.

Reply:
Fixed